packages/prerender-proxy/lib/handlers/cache-control.ts

```diff
@@ -1,21 +1,8 @@
-import 'source-map-support/register';
-import { CloudFrontResponseEvent, CloudFrontResponse  } from 'aws-lambda';
-/*  
-    Prerender cache control function 
-    Consider associate this function as *origin response* function
-*/
-export const handler = async (event: CloudFrontResponseEvent): Promise<CloudFrontResponse> => {
-    const cacheKey = process.env.PRERENDER_CACHE_KEY || 'x-prerender-requestid';
-    const cacheMaxAge = process.env.PRERENDER_CACHE_MAX_AGE || '0';
-    let response = event.Records[0].cf.response;
+import middy from "@middy/core";
+import { PrerenderCacheControlHandler } from "@aligent/cdk-lambda-at-edge-handlers";
 
-    if (response.headers[`${cacheKey}`]) {
-        response.headers['Cache-Control'] = [
-            {
-                key: 'Cache-Control',
-                value: `max-age=${cacheMaxAge}`
-            }
-        ]
-    }
-    return response;
-}
+/**
+ * Prerender Cache Control Handler
+ * @link https://github.com/aligent/cdk-constructs/blob/main/packages/lambda-at-edge-handlers/lib/cache-control.ts
+ */
+export const handler = middy(PrerenderCacheControlHandler);
```

Comment:
Instead of returning the functions as a middy wrapper, perhaps we must make this into a middy middleware instead? That means end user can make their own base functions and wrap it with middy, and then call `.use(prerenderMiddleWare)`